JitsiConference.js

```diff
@@ -579,22 +579,27 @@ JitsiConference.prototype.removeCommandListener = function(command) {
 /**
  * Sends text message to the other participants in the conference
  * @param message the text message.
+ * @param elementName the element name to encapsulate the message.
  * @deprecated Use 'sendMessage' instead. TODO: this should be private.
  */
-JitsiConference.prototype.sendTextMessage = function(message) {
+JitsiConference.prototype.sendTextMessage = function(
+        message, elementName = 'body') {
     if (this.room) {
-        this.room.sendMessage(message);
+        this.room.sendMessage(message, elementName);
     }
 };
 
 /**
  * Send private text message to another participant of the conference
+ * @param id the id of the participant to send a private message.
  * @param message the text message.
+ * @param elementName the element name to encapsulate the message.
  * @deprecated Use 'sendMessage' instead. TODO: this should be private.
  */
-JitsiConference.prototype.sendPrivateTextMessage = function(id, message) {
+JitsiConference.prototype.sendPrivateTextMessage = function(
+        id, elementName = 'body', message) {
```

Comment:
Why is `elementName` the second parameter and not the third? I expect it to be third because:
1. It has a default value, so it's natural to leave it last
2. In `sendTextMessage` the `elementName` comes after `message`